### DISCUSSION TITLE
Nuget package backward compatibility with Xamarin.Form

### DISCUSSION THREAD

**Discussion by dominic7125** [CommunityToolkit/Maui#360] [Q&A] [answered]
We have lots of customers are still using Xamarin.Form.
Right now, we are trying to miragte our XF nuget to MAUI, but it's painful if we'll need to maintain two frameworks.
Is there a way to upgrade our existing XF nuget package to support MAUI and XF at same time?

**Comment by bijington** [comment]
@dominic7125 I think this is more likely a question for the .NET MAUI team. Their repo is over at: https://github.com/dotnet/maui

**Comment by VladislavAntonyuk** [comment, marked as answer]
Hi, you can see how we migrate Xamarin Community Toolkit to Xamarin Community Toolkit MauiCompat: https://github.com/xamarin/XamarinCommunityToolkit/blob/main/MauiCompatSteps.md

**Comment by CliffAgius** [comment]
Closed as answered
